packages/shorebird_cli/lib/src/patch_diff_checker.dart

```diff
@@ -67,7 +69,8 @@ class PatchDiffChecker {
             archiveDiffer.nativeFileSetDiff(contentDiffs).prettyString,
           ),
         );
-      final shouldContinue = force || logger.confirm('Continue anyways?');
+      final shouldContinue =
```

Comment:
is this logic correct? I thought we wanted this to exit in CI instead of continuing.

Reply:
That's what it does. `shouldConfirm` is basically "we're not on CI". If the user doesn't pass the `--force` flag and `shouldConfirm` is false, `shouldContinue` will be false. Reworked a bit for clarity.

Reply:
This line is also a bit dense – I wouldn't be opposed to further refactoring it out into a few separate checks.